src/main.ts

```diff
@@ -257,7 +257,9 @@ export class Minifier {
       // Property names cannot start with a number. Generate next possible property name that starts
       // with the first alpha character.
     } else if (chars[0].match(/[0-9]/)) {
-      return (firstAlpha + Array(len).join(firstChar));
+      newName = firstAlpha + Array(len).join(firstChar);
+      this.lastGeneratedPropName = newName;
```

Comment:
This points to a bigger issue with this function: you always need to do set lastGenerated before returning
```
this.lastGeneratedPropName = newName;
return newName;
```

Can you think of a way to guarantee that if someone adds another early `return` statement in this function they don't end up with this bug again?